app/src/main/java/com/example/tung/moviedb_07/screen/search_result/SearchResultViewModel.java

```diff
@@ -0,0 +1,21 @@
+package com.example.tung.moviedb_07.screen.search_result;
+
+import com.example.tung.moviedb_07.R;
+import com.example.tung.moviedb_07.screen.BaseViewModel;
+
+/**
+ * Created by tung on 6/8/17.
+ */
+
+public class SearchResultViewModel extends BaseViewModel {
+
+    private String mSearchKeyword;
+
+    public SearchResultViewModel(String searchKeyword) {
+            mSearchKeyword = searchKeyword;
```

Comment:
ERROR: 'ctor def' child have incorrect indentation level 12, expected level should be 8.